daisy/error_test.go

```diff
@@ -124,3 +125,15 @@ func TestDErrImplAdd(t *testing.T) {
 		}
 	}
 }
+
+func TestNestedAnonymizedDErrorMessage(t *testing.T) {
+	innerDErr1 := Errf("inner error 1: %v %v", "root cause 1", "root cause 2")
+	innerDErr2 := Errf("inner error 2: %v %v", "root cause 3", "root cause 4")
+	innerDErr1.add(innerDErr2)
+	outerDErr := wrapErrf(innerDErr1, "outer error: %v: ", "bad news")
+	got := strings.Join(outerDErr.AnonymizedErrs(), ",")
+	want := "outer error: %v: inner error 1: %v %v; inner error 2: %v %v"
```

Comment:
Why aren't placeholders (%v) replaced with values in the resulting error here ("root cause 1", ..."bad news") etc.?

Reply:
It's anonymized msg. It's designed to hide the concrete value of placeholder.